### ISSUE TITLE
Two problems. relaunch() and signing on win7

### ISSUE THREAD

**Issue by Alex313031** [castlabs/electron-releases#164]
- Leaving code identical and simply switching out upstream electron for castlabs causes app.relaunch(); followed by app.quit(); to not 
relaunch the app on Linux or Windows 7. 

VMS signing on Windows 7 doesn't seem to work. Using Python 3.9 just like on my Win10 machine where singing works.

**Comment by khwaaj** [comment]
Which version of ECS are you using? Also, do you have some logs from when VMP signing fails?

**Comment by Alex313031** [comment]
@khwaaj Fixed the VMP signing, it was an error on my part.

However, the relaunch issue is still present on all platforms. 

Using vanilla electron works fine. I made no changes to my code or the electron version, only whether it was vanilla vs Castlabs. The issue occurs with Electron 21 - 25. Seems to be resolved with 26, but I am purposefully using Electron 22 for Win7/8/8.1 and MacOS <10.14 support.

**Comment by khwaaj** [comment]
This is puzzling indeed. None of the changes we have made to Electron even remotely relate to relaunch behavior, so it is very unclear what could be the culprit here.

We did have a similar issue reported a while back which I was not able to reproduce in my setup, I'll see about reproducing this one. Do you perhaps have a minimal example to share?

**Comment by Alex313031** [comment]
@khwaaj https://github.com/Alex313031/castlabs-test

**Comment by khwaaj** [comment]
Just tested your minimal example with `v25.7.0+wvcus-darwin-arm64` (Ventura) and I can't reproduce the issue. Selecting "Restart App" in the menu quits and relaunches the application as expected.

You said "all platforms", but can this be limited somehow to the setups you are using?

**Comment by Alex313031** [comment]
@khwaaj I'm sorry, I shouldn't have said all platforms. I don't have a Mac, so I couldn't test there. On linux it definitely shows up. And I dont think its my system. I have tried on ubuntu and debian, with node 14, 16, and 18, and with various castlabs releases from 21-15

**Comment by Alex313031** [comment]
@khwaaj Are you there?

**Comment by khwaaj** [comment]
Absolutely, there are just a lot of other things going on that has priority at the moment, so I'm not entirely sure when there will be time to debug this properly.

**Comment by Alex313031** [comment]
@khwaaj Issue is still present in Electron 28

**Comment by Alex313031** [comment]
@khwaaj Issue is still present in all Electron versions including 29 and latest beta. No response since september of last year, and all my apps that have a "restart" or "relaunch" function fail to do so on Linux. I get github issues all the time with users asking why it doesn't work and I have to explain that it's an upstream thing. It's getting frustrating.

**Comment by khwaaj** [comment]
I did look into this a bit more recently, and I am able to reproduce it on Linux (but nowhere else). Unfortunately the cause is not clear so there is no proper solution yet, it does however seem related to a bunch of other issues reported for ECS as well as other Electron based projects (all of these involve some combination of SIGTRAP/SIGILL errors during sandbox/zygote startup). Passing `--no-sandbox --no-zygote` arguments does fix the relaunch issue in my test, albeit is not a good production solution.
